src/zimlet/com_zimbra_birthdayreminder/com_zimbra_birthdayreminder.properties

```diff
@@ -38,7 +38,7 @@ BirthdayReminder_formatIncorrect = Birthdays must have "YYYY-MM-DD" format
 # Preferences dialog
 #
 BirthdayReminder_zimletPreferences = Birthday Reminder: Preferences
-BirthdayReminder_PleaseNote = Note: This Zimlet automatically creates a Calendar called <b>'{0}'</b> <br/> to store all birthday reminders.
+BirthdayReminder_PleaseNote = Note: This Zimlet automatically creates a Calendar called <b>''{0}''</b> <br/> to store all birthday reminders.
```

Comment:
should we use smart quotes?

Reply:
will be fixed with ZCS-2592 in all files

Reply:
thanks